invoke.bat

```diff
@@ -0,0 +1,10 @@
+@echo off
+
+set INSTALL_ENV_DIR=%cd%\installer_files\env
+set PATH=%INSTALL_ENV_DIR%;%INSTALL_ENV_DIR%\Library\bin;%INSTALL_ENV_DIR%\Scripts;%INSTALL_ENV_DIR%\Library\usr\bin;%PATH%
+
+call conda activate invokeai
+
+echo Ready to dream..
```

Comment:
I'd like to see both this and the shell script *at least* print the command  to launch Invoke (instead of "Ready to dream..") - otherwise I can promise you that people are going to be complaining that they "launched invoke" (meaning ran this script), but aren't able to generate images.

**Ideally**, we should present a choice (e.g. "Run CLI or Web?") and actually launch Invoke according to the user's preference.